couler/tests/mpi_step_test.py

```diff
@@ -0,0 +1,82 @@
+from io import StringIO
+
+import yaml
+
+import couler.argo as couler
+import couler.steps.mpi as mpi
+from couler.core import utils
+from couler.tests.argo_yaml_test import ArgoYamlTest
+
+
+class MPITestCase(ArgoYamlTest):
+    def test_mpi_train(self):
+        access_key_secret = {"access_key": "key1234"}
+        secret = couler.create_secret(secret_data=access_key_secret)
+
+        mpi.train(
+            num_workers=3,
+            image="mpi:1.13",
+            command="python mpi.py",
+            worker_resources="cpu=0.5,memory=1024",
+            clean_pod_policy="Running",
+            secret=secret,
```

Comment:
how can we use a secret here ? from the env or others?

Reply:
This may be unnecessary and need to be double checked. I'll revisit this later in a separate PR as it appears in other pre-defined steps as well.